sdk-libs/halo-social/proguard-rules.pro

```diff
@@ -0,0 +1,7 @@
+-keepattributes Exceptions
+-keepparameternames
+-keepattributes EnclosingMethod
+
+-keep class com.bluelinelabs.logansquare.** { *; }
+-keep @com.bluelinelabs.logansquare.annotation.JsonObject class *
```

Comment:
According to official documentation pages there is no need to add proguard rules:
https://developers.facebook.com/docs/android/getting-started
https://developers.google.com/android/guides/setup